ui/src/mini-cart-module/mini-cart-page/mini-cart-page.component.sass

```diff
@@ -0,0 +1,141 @@
+
+.cart-container
+    display: flex
+    flex-direction: column
+    height: 100%
+    margin-top: 0
+
+
+.modal-header
+    display: flex
+    justify-content: space-between
+    background-color: black
+    color: #fff
+    align-items: center
+    padding: 5px
+
+@media (min-width: 1024px)
+    .modal-header h2
+        font-size: 20px
+
+.close
+    color: #fff
+    float: right
+    background: transparent
+    font-size: 15px
+    font-weight: bold
+    margin-right: -10px
+
+.close:hover,.close:focus
+    cursor: pointer
+
+.cart-items-warapper
+    height: 70%
+    width: 100%
+    box-shadow: 0 8px 6px -6px #d8d8d8
+
+li:not(.cart-footer-banner)
+    display: flex
+    justify-content: space-between
+    align-items: center
+    margin: 10px
+    padding: 10px
+    background-color: #fff
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2)
+    transition: 0.3s
+
+.cart-item-image
+    height: 60px
+    width: 60px
+    margin-left: 10px
+
+img.product-card-img
+    background: white
+    width: 100%
+    height: 100%
+    position: relative
+    overflow: hidden
+
+.cart-item-desc
+    flex: 1
+    margin-left: 10px
+
+.cart-item-title
+    font-weight: 600
+
+li:not(.cart-footer-banner) .btn
+    border-radius: 40%
+    height: 25px
+    width: 23px
+    padding: 1px 5px
+    margin: 5px
+
+.cart-item-total
+    text-align: right
+    margin-right: 15px
+
+.cart-footer-banner
+    height: 40px
+    display: flex
+    width: 100%
+    background-color: #fff
+    justify-content: space-evenly
+    align-items: center
+    margin-bottom: 10px
+    padding: 10px
+
+
+.cart-footer
+    text-align: center
+    background: #fff
+    padding: 5px 15px
+    width: 100%
+
+@media (min-width: 1024px)
```

Comment:
Common media breakpoints should be put into SCSS/SASS variables, and can be used as mix-ins (all instances)

Reply:
Done